src/app/core/allocation/allocation-preview.service.ts

```diff
@@ -32,9 +32,13 @@ export class AllocationPreviewService {
         endAt,
         totalGoal,
         dailyMinimum,
-        zones
+        zones: this.getZoneIds(zones)
       });
     }
     return preview;
   }
+
+  private getZoneIds(zones: any[]): string[] {
```

Comment:
maybe make this a static function in the flight.model (and filter before the map)